packages/config/src/lib/environments/environment.prod.ts

```
@@ -193,6 +193,8 @@ export const environment: IEnvironment = {
 	zapier: {
 		clientId: process.env.GAUZY_ZAPIER_CLIENT_ID,
 		clientSecret: process.env.GAUZY_ZAPIER_CLIENT_SECRET,
+		allowedDomains: process.env.GAUZY_ALLOWED_DOMAINS ? process.env.GAUZY_ALLOWED_DOMAINS.split(',').map(domain => domain.trim()) : [],
+		maxAuthCodes: parseInt(process.env.MAX_AUTH_CODES) || 1000,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)